### ISSUE TITLE
Calculate rmsDb in AudioRecorder

### ISSUE THREAD

**Issue by morfeusys** [just-ai/aimybox-android-sdk#37]
`AudioRecorder` class is used in some speech to text modules. It would be great to have a single function that calculates a speech volume level because `AimyboxButton` UI component can visualise a sound level changes.

It's good starting point about RMSdb [here](https://stackoverflow.com/questions/40235122/android-audiorecord-pcm-to-get-db).

**Comment by morfeusys** [comment]
Here is a working example
```
    private static int calculateRMSLevel(byte[] data) {
        long lSum = 0;
        for (byte aData1 : data) lSum = lSum + aData1;

        double dAvg = lSum / data.length;

        double sumMeanSquare = 0d;
        for (byte aData : data) sumMeanSquare = sumMeanSquare + Math.pow(aData - dAvg, 2d);

        double averageMeanSquare = sumMeanSquare / data.length;
        return (int)(Math.pow(averageMeanSquare, 0.5d) + 0.5);
    }
```

**Comment by morfeusys** [comment]
Released in 0.7.0
